README.md

```diff
@@ -37,6 +37,19 @@ The *GoodData Java SDK* uses:
 * the *Slf4j API* version 1.7.*
 * the *Java Development Kit (JDK)* version 8 or later
 
+##### Retry of failed API calls
+
+Since version *2.33.1* you can retry your failed requests. Turn it on by configuring
```

Comment:
What about to raise minor version with this new feature? It's not a bug fix ;)

Reply:
OK :-)